app/Util/DocuSign.php

```diff
@@ -366,4 +381,421 @@ public static function membershipAgreementEnvelopeDefinition(DocuSignEnvelope $e
+     */
+    private static function directBillAirfareFormDocument(TravelAssignment $assignment): Document
+    {
+        $textTabs = [
+            (new Text())
+                ->setTabType('text')
+                ->setDocumentId(2)
+                ->setPageNumber(1)
+                ->setFont('CourierNew')
+                ->setFontColor('Black')
+                ->setFontSize('Size12')
+                ->setHeight(20)
+                ->setWidth(100)
+                ->setXPosition(160)
+                ->setYPosition(235)
+                ->setValue($assignment->user->phone),
+            (new Text())
+                ->setTabType('text')
+                ->setDocumentId(2)
+                ->setPageNumber(1)
+                ->setFont('CourierNew')
+                ->setFontColor('Black')
+                ->setFontSize('Size12')
+                ->setHeight(20)
+                ->setWidth(100)
+                ->setXPosition(185)
+                ->setYPosition(260)
+                ->setValue($assignment->travel->tar_purpose),
+            (new Text())
+                ->setTabType('text')
+                ->setDocumentId(2)
+                ->setPageNumber(1)
+                ->setFont('CourierNew')
+                ->setFontColor('Black')
+                ->setFontSize('Size12')
+                ->setHeight(20)
+                ->setWidth(100)
+                ->setXPosition(160)
+                ->setYPosition(370)
+                ->setValue($assignment->travel->tar_project_number),
+        ];
+
+        if ($assignment->user->employee_id !== null) {
+            $textTabs[] = (new Text())
+                ->setTabType('text')
+                ->setDocumentId(2)
+                ->setPageNumber(1)
+                ->setFont('CourierNew')
+                ->setFontColor('Black')
+                ->setFontSize('Size12')
+                ->setHeight(20)
+                ->setWidth(100)
+                ->setXPosition(160)
+                ->setYPosition(435)
+                ->setValue($assignment->user->employee_id);
+        }
+
+        return (new Document())
+            ->setDocumentId(2)
+            ->setDisplay('inline')
+            ->setDocumentBase64(base64_encode(file_get_contents(resource_path('pdf/direct_bill_airfare_form.pdf'))))
+            ->setIncludeInDownload(true)
+            ->setName('Request for Single-Trip Direct Billing of Airfare')
+            ->setTabs(
+                (new Tabs())
+                    ->setPrefillTabs(
+                        (new PrefillTabs())
+                            ->setTextTabs($textTabs)
+                    )
+            );
+    }
+
+    /**
+     * Builds the array of documents for this travel assignment.
+     *
+     * @return array<\DocuSign\eSign\Model\Document>
+     */
+    private static function travelAssignmentDocuments(TravelAssignment $assignment): array
+    {
+        $documents = [];
+
+        if ($assignment->travel->tar_required === true) {
```

Comment:
is there a reason to leave `=== true` here?

```suggestion
        if ($assignment->travel->tar_required) {
```

Reply:
`tar_required` is technically nullable